src/hotspot/share/opto/compile.hpp

```diff
@@ -618,6 +619,8 @@ class Compile : public Phase {
   void          set_print_inlining(bool z)       { _print_inlining = z; }
   bool              print_intrinsics() const     { return _print_intrinsics; }
   void          set_print_intrinsics(bool z)     { _print_intrinsics = z; }
+  bool              trace_escape_analysis() const{ return _trace_escape_analysis; }
```

Comment:
You can skip the setter and getter and just use compile->directive()->TraceEscapeAnalysisOption at the use sites.